db_scripts/create_tables.sql

```diff
@@ -95,8 +95,173 @@ CREATE TABLE socceroffsides (
+    starttime TIMESTAMP NOT NULL DEFAULT NOW(),
+
+    CONSTRAINT fk_basketball_league FOREIGN KEY(leagueId) REFERENCES leagues(id)
+);
+
+-- postgres automatically indexes primary key + unique columns.
+-- indexing PK just to be consistent with the previous DDL statements
+CREATE INDEX IF NOT EXISTS basketball_pkey ON basketballgames(id uuid_ops);
 
 
+------------------------------------------------------------------
+
+CREATE TABLE basketballgameevents (
+    id uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    gameId uuid NOT NULL,
+    playType CHARACTER VARYING(255) NOT NULL CHECK (playType in (
+        '1pt',
+        '2pt',
+        '3pt',
+        'foul',
+        'turnover',
+        'timeout',
+        'end',
+        'block',
+        'steal',
+        'rebound'
+    )),
+    period SMALLINT NOT NULL CHECK (period in (1,2,3,4)),
+    teamFor CHARACTER VARYING(255),
+    teamAgainst CHARACTER VARYING(255),
+
+    CONSTRAINT fk_basketball_events_game FOREIGN KEY(gameId) REFERENCES basketballgames(id)
+);
+
+CREATE INDEX IF NOT EXISTS basketball_events_pkey ON basketballgameevents(id uuid_ops);
+CREATE INDEX IF NOT EXISTS basketball_events_playtype ON basketballgameevents(playType text_ops);
+CREATE INDEX IF NOT EXISTS basketball_events_period ON basketballgameevents(period int2_ops);
+
+------------------------------------------------------------------
+
+CREATE TABLE basketballonept (
+    id uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    eventId uuid NOT NULL,
+    player CHARACTER VARYING(255) NOT NULL,
+    result CHARACTER VARYING(255) NOT NULL CHECK (result in (
+        'made',
+        'miss'
+    )),
+
+    CONSTRAINT fk_basketball_1pt_event FOREIGN KEY(eventId) REFERENCES basketballgameevents(id)
+);
 
+CREATE INDEX IF NOT EXISTS basketball_events_1pt_pkey ON basketballonept(id uuid_ops);
+
+------------------------------------------------------------------
+
+CREATE TABLE basketballtwopts (
+    id uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    eventId uuid NOT NULL,
+    player CHARACTER VARYING(255) NOT NULL,
+    assist CHARACTER VARYING(255),
+    result CHARACTER VARYING(255) NOT NULL CHECK (result in (
+        'made',
+        'miss'
+    )),
+
+    CONSTRAINT fk_basketball_2pts_event FOREIGN KEY(eventId) REFERENCES basketballgameevents(id)
+);
+
+CREATE INDEX IF NOT EXISTS basketball_events_2pts_pkey ON basketballtwopts(id uuid_ops);
+
+------------------------------------------------------------------
+
+CREATE TABLE basketballthreepts (
+    id uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    eventId uuid NOT NULL,
+    player CHARACTER VARYING(255) NOT NULL,
+    assist CHARACTER VARYING(255),
+    result CHARACTER VARYING(255) NOT NULL CHECK (result in (
+        'made',
+        'miss'
+    )),
+
+    CONSTRAINT fk_basketball_3pts_event FOREIGN KEY(eventId) REFERENCES basketballgameevents(id)
+);
+
+CREATE INDEX IF NOT EXISTS basketball_events_3pts_pkey ON basketballthreepts(id uuid_ops);
```

Comment:
Consolidate into one schema: 

![image](https://user-images.githubusercontent.com/38513862/220198705-643a2732-e5a7-40ad-90eb-7de3f4517399.png)

Reply:
ack

Reply:
@KnotScientific @sid42 I made the requested changes. Ready for re-review